functions/src/index.ts

```diff
@@ -38,15 +22,45 @@ userApp.get('/users/:userId/', (request, response) => {
   response.redirect("/users/" + request.params.userId + "/invites" )
 })
 
-userApp.post('/users/:userId/invites', (request: RequestWithAuth, response) => {
-  if (!request.auth || !request.auth.uid) {
```

Comment:
It turned out that `request.auth` is not available inside Firebase functions. That is only available in Firestore rules. Bah.